### PR TITLE
Remove bitwise operators and support for hexadecimal values.

### LINKED ISSUE #15: Remove bitwise operators and support of hexadecimal values
During ADASS and the IVOA Interoperability Meeting of Oct. 2019, it has been decided to not tackle bitwise operators and hexadecimal values in ADQL-2.1. The current description of these features introduce too many differences or issues in implementations (especially with MySQL). 

_For more details, see from slides 26 of the talk ["Feedback on the implementation of ADQL-2.1"](https://wiki.ivoa.net/internal/IVOA/InterOpOct2019DAL/feedback_adql2_1.pdf)._

These features are not yet mature. More thoughts are required. They may be introduced again in a next version of ADQL.

### PR DESCRIPTION
**Pull request by gmantele**
Fixes #15

### CODE REVIEW

**Comment by gmantele** [comment]
Review still required on this PR about removal of bitwise operators.
Is there anybody among watchers and @msdemlei wanting to take a look?

File: ADQL.tex [hunk 1]
```diff
@@ -348,17 +348,15 @@ \subsubsection{Literals}
         <quote> [ <character_representation>... ] <quote>
 \end{verbatim}
 
-Numeric literals may be are expressed as an exact decimal value, e.g. \verb:12: or \verb:12.3:,
-a floating point number with an exponent, e.g. \verb:12.3E4:, or a unsigned hexadecimal
-value, e.g. \verb:0x2F:.
+Numeric literals are expressed as an exact decimal value, e.g. \verb:12: or
+\verb:12.3:, a floating point number with an exponent, e.g. \verb:12.3E4:.
```
Comment (jd-au) [inline ADQL.tex]: Should have an or in here "or a floating..."

**Review by msdemlei — DISMISSED** [review]
Apart from the grammar fix James suggested (yeah, it should be an "or"), I'm all in favour of this.

File: ADQL.tex (hunk 1 above)
Comment (gmantele) [reply ADQL.tex]: Fixed

**Review by jd-au — DISMISSED** [review]
The changes look good, but the BNF definition of <hex_digit> in appendix A could also be removed now I think.

**Comment by gmantele** [comment]
> The changes look good, but the BNF definition of <hex_digit> in appendix A could also be removed now I think.

@jd-au , I agree. I will do that before merging the PR.

**Comment by gmantele** [comment]
The CI does not work anymore. Apparently, the step 'Setup dependencies' fails for some reason linked to the linux distribution:

```
W: Conflicting distribution: https://packages.microsoft.com/repos/microsoft-ubuntu-bionic-prod bionic InRelease (expected bionic but got stable)
E: Repository 'https://packages.microsoft.com/repos/microsoft-ubuntu-bionic-prod bionic InRelease' changed its 'Origin' value from 'microsoft-ubuntu-bionic-prod bionic' to 'vscode stable'
E: Repository 'https://packages.microsoft.com/repos/microsoft-ubuntu-bionic-prod bionic InRelease' changed its 'Label' value from 'microsoft-ubuntu-bionic-prod bionic' to 'vscode stable'
E: Repository 'https://packages.microsoft.com/repos/microsoft-ubuntu-bionic-prod bionic InRelease' changed its 'Suite' value from 'bionic' to 'stable'
E: Repository 'https://packages.microsoft.com/repos/microsoft-ubuntu-bionic-prod bionic InRelease' changed its 'Codename' value from 'bionic' to 'stable'
W: Conflicting distribution: https://packages.microsoft.com/ubuntu/16.04/prod xenial InRelease (expected xenial but got stable)
E: Repository 'https://packages.microsoft.com/ubuntu/16.04/prod xenial InRelease' changed its 'Origin' value from 'microsoft-ubuntu-xenial-prod xenial' to 'vscode stable'
E: Repository 'https://packages.microsoft.com/ubuntu/16.04/prod xenial InRelease' changed its 'Label' value from 'microsoft-ubuntu-xenial-prod xenial' to 'vscode stable'
E: Repository 'https://packages.microsoft.com/ubuntu/16.04/prod xenial InRelease' changed its 'Suite' value from 'xenial' to 'stable'
E: Repository 'https://packages.microsoft.com/ubuntu/16.04/prod xenial InRelease' changed its 'Codename' value from 'xenial' to 'stable'
##[error]Process completed with exit code 100.
```

Any idea?

**Review by jd-au — APPROVED** [review]
All looks good. The CI failure was temporary outage. I've rerun them now and it has passed.

**Pull request merged**